mars/dataframe/base/eval.py

```diff
@@ -123,7 +123,7 @@ def get_named_object(self, obj_name):
         raise KeyError(f'name {obj_name} is not defined')
 
     def visit(self, node):
-        if isinstance(node, DATAFRAME_TYPE):
+        if isinstance(node, DATAFRAME_TYPE) or isinstance(node, SERIES_TYPE):
```

Comment:
We already have an `ENTITY_TYPE` for all Mars objects. Use that type will be better.